float32 target_value
---
bool is_succeed